tools/maf_clean_up.pl

```diff
@@ -0,0 +1,181 @@
+			} elsif (uc($columns[$keep_col_index]) eq "6-MAR") {
+				print DAT "MARCH6";
+			} elsif (uc($columns[$keep_col_index]) eq "7-MAR") {
+				print DAT "MARCH7";
+			} elsif (uc($columns[$keep_col_index]) eq "8-MAR") {
+				print DAT "MARCH8";
+			} elsif (uc($columns[$keep_col_index]) eq "9-MAR") {
+				print DAT "MARCH9";
+			} elsif (uc($columns[$keep_col_index]) eq "10-MAR") {
+				print DAT "MARCH10";
+			} elsif (uc($columns[$keep_col_index]) eq "11-MAR") {
+				print DAT "MARCH11";
+			} elsif (uc($columns[$keep_col_index]) eq "1-APR") {
+				print DAT "MAGEH1";
+			} elsif (uc($columns[$keep_col_index]) eq "2-APR") {
+				print DAT "FAM215A";
+			} elsif (uc($columns[$keep_col_index]) eq "3-APR") {
+				print DAT "ATRAID";
+			} elsif (uc($columns[$keep_col_index]) eq "1-MAY") {
+				print DAT "PRKCD";
+			} elsif (uc($columns[$keep_col_index]) eq "1-SEP") {
+				print DAT "SEPT1";
+			} elsif (uc($columns[$keep_col_index]) eq "2-SEP") {
+				print DAT "SEPT2";
+			} elsif (uc($columns[$keep_col_index]) eq "3-SEP") {
+				print DAT "SEPT3";
+			} elsif (uc($columns[$keep_col_index]) eq "4-SEP") {
+				print DAT "SEPT4";
+			} elsif (uc($columns[$keep_col_index]) eq "5-SEP") {
+				print DAT "SEPT5";
+			} elsif (uc($columns[$keep_col_index]) eq "6-SEP") {
+				print DAT "SEPT6";
+			} elsif (uc($columns[$keep_col_index]) eq "7-SEP") {
+				print DAT "SEPT7";
+			} elsif (uc($columns[$keep_col_index]) eq "8-SEP") {
+				print DAT "SEPT8";
+			} elsif (uc($columns[$keep_col_index]) eq "9-SEP") {
+				print DAT "SEPT9";
+			} elsif (uc($columns[$keep_col_index]) eq "10-SEP") {
+				print DAT "SEPT10";
+			} elsif (uc($columns[$keep_col_index]) eq "11-SEP") {
+				print DAT "SEPT11";
+			} elsif (uc($columns[$keep_col_index]) eq "12-SEP") {
+				print DAT "SEPT12";
+			} elsif (uc($columns[$keep_col_index]) eq "13-SEP") {
+				print DAT "SEPT13";
+			} elsif (uc($columns[$keep_col_index]) eq "14-SEP") {
+				print DAT "SEPT14";
+			} elsif (uc($columns[$keep_col_index]) eq "2-OCT") {
+				print DAT "POU2F2";
+			} elsif (uc($columns[$keep_col_index]) eq "2-OCT") {
+				print DAT "SLC22A2";
+			} elsif (uc($columns[$keep_col_index]) eq "3-OCT") {
+				print DAT "POU5F1";
+			} elsif (uc($columns[$keep_col_index]) eq "4-OCT") {
+				print DAT "POU5F1";
+			} elsif (uc($columns[$keep_col_index]) eq "6-OCT") {
+				print DAT "POU3F1";
+			} elsif (uc($columns[$keep_col_index]) eq "6-OCT") {
+				print DAT "SLC22A16";
+			} elsif (uc($columns[$keep_col_index]) eq "7-OCT") {
+				print DAT "POU3F2";
+			} elsif (uc($columns[$keep_col_index]) eq "9-OCT") {
+				print DAT "POU3F4";
+			} elsif (uc($columns[$keep_col_index]) eq "11-OCT") {
+				print DAT "POU2F3";
+			} elsif (uc($columns[$keep_col_index]) eq "1-NOV") {
+				print DAT "C11orf40";
+			} elsif (uc($columns[$keep_col_index]) eq "2-NOV") {
+				print DAT "CTGF";
+			} elsif (uc($columns[$keep_col_index]) eq "1-DEC") {
+				print DAT "DEC1";
+			}
+		} 
+		$col_counter++;
```

Comment:
Reference comment above where the loop isn't continuing to next iteration even though we may have already printed the corrected Hugo_Symbol. At the end of the if/else blocks below this, we end up printing the value as long as it's not "START_POSITION", "END_POSITION", "HUGO_SYMBOL".